R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2083,6 +2083,43 @@ test_that("dapply() on a DataFrame", {
   expect_identical(expected, result)
 })
 
+test_that("gapply() on a DataFrame", {
```

Comment:
Added a new test which was used for our previous group-apply showcase for customers.